[2025-08-02T22:07:21.447+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-02T22:07:21.477+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_process_all_files_v1.move_users_file manual__2025-08-02T22:07:15.539522+00:00 [queued]>
[2025-08-02T22:07:21.489+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_process_all_files_v1.move_users_file manual__2025-08-02T22:07:15.539522+00:00 [queued]>
[2025-08-02T22:07:21.492+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-02T22:07:21.513+0000] {taskinstance.py:2330} INFO - Executing <Task(GCSToGCSOperator): move_users_file> on 2025-08-02 22:07:15.539522+00:00
[2025-08-02T22:07:21.527+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1183) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-02T22:07:21.530+0000] {standard_task_runner.py:63} INFO - Started process 1192 to run task
[2025-08-02T22:07:21.531+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'gcs_process_all_files_v1', 'move_users_file', 'manual__2025-08-02T22:07:15.539522+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/gcs_load_dag v2.py', '--cfg-path', '/tmp/tmptjr8kj0g']
[2025-08-02T22:07:21.533+0000] {standard_task_runner.py:91} INFO - Job 53: Subtask move_users_file
[2025-08-02T22:07:21.638+0000] {task_command.py:426} INFO - Running <TaskInstance: gcs_process_all_files_v1.move_users_file manual__2025-08-02T22:07:15.539522+00:00 [running]> on host 06d424bf0ee1
[2025-08-02T22:07:21.784+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gcs_process_all_files_v1' AIRFLOW_CTX_TASK_ID='move_users_file' AIRFLOW_CTX_EXECUTION_DATE='2025-08-02T22:07:15.539522+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-02T22:07:15.539522+00:00'
[2025-08-02T22:07:21.787+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-02T22:07:21.803+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-08-02T22:07:21.812+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2025-08-02T22:07:23.066+0000] {gcs_to_gcs.py:534} INFO - Executing copy of gs://dataverse360-raw-data-rmk/landing/users_data.csv to gs://dataverse360-raw-data-rmk/raw/users_data.csv
[2025-08-02T22:07:23.284+0000] {gcs.py:261} INFO - Total Bytes: 164831 | Bytes Written: 164831
[2025-08-02T22:07:23.285+0000] {gcs.py:269} INFO - Object landing/users_data.csv in bucket dataverse360-raw-data-rmk rewritten to object raw/users_data.csv in bucket dataverse360-raw-data-rmk
[2025-08-02T22:07:23.286+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-02T22:07:23.304+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=gcs_process_all_files_v1, task_id=move_users_file, run_id=manual__2025-08-02T22:07:15.539522+00:00, execution_date=20250802T220715, start_date=20250802T220721, end_date=20250802T220723
[2025-08-02T22:07:23.353+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-02T22:07:23.394+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-02T22:07:23.397+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
